website/static/js/pages/project-dashboard-page.js

```diff
@@ -93,6 +94,32 @@ var institutionLogos = {
     }
 };
 
+/* Send with ajax calls to work with api2 */
+var xhrconfig = function (xhr) {
```

Comment:
We recently added a utility function to generate xhr config functions for APIv2. If you pull from the latest develop, you'll be able to use `mithrilHelpers.apiV2Config`.